static/app/views/organizationGroupDetails/header.tsx

```diff
@@ -275,7 +275,7 @@ function GroupHeader({
         {group.issueType === IssueType.PERFORMANCE_CONSECUTIVE_DB_QUERIES && (
           <FeatureBadge
             type="alpha"
-            title="Slow DB Span Performance Issues are in active development and may change"
+            title="Consecutive DB Query Performance Issues are in active development and may change"
```

Comment:
Can we wrap this in `t()` for translations?

Reply:
Sure, I'll open another PR for that 👍